ios/sdk/src/analytics/AmplitudeModule.m

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright @ 2018-present 8x8, Inc.
+ * Copyright @ 2017-2018 Atlassian Pty Ltd
```

Comment:
ditto, only the 8x8 header, and starting in 2019, which is when this file was added.